easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,163 @@
+            {'AlphaFold-2.3.2_use_openmm_8.0.0.patch':
+             'bbef940c0c959040aaf3984ec47777a229c164517b54616a2688d58fae636d84'},
+            {'AlphaFold-2.3.2_BioPythonSCOPData.patch':
+             '5aedd5a35da6d0a993ca19f4de807cf77ec36a5a52c97100110a0c5475cef1b8'},
+        ],
+    }),
+]
+
+local_pylibdir = '%(installdir)s/lib/python%(pyshortver)s/site-packages'
+local_link_scp = 'ln -s %%(installdir)s/stereo_chemical_props.txt %s/alphafold/common' % local_pylibdir
+
+postinstallcmds = [
+    'cp %(builddir)s/AlphaFold/alphafold-%(version)s/run_alphafold*.py %(installdir)s/bin',
+    'cp -rpP %(builddir)s/AlphaFold/alphafold-%(version)s/scripts %(installdir)s',
+    'cd %(installdir)s/bin && ln -s run_alphafold.py alphafold',
+    'chmod a+x %(installdir)s/bin/run_alphafold.py',
+    local_link_scp,
+]
+
+sanity_check_paths = {
+    'files': ['bin/alphafold', 'bin/pdbfixer', 'bin/run_alphafold.py', 'stereo_chemical_props.txt'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages', 'scripts'],
+}
+
+sanity_check_commands = [
+    "pdbfixer --help",
+    "python -m openmm.testInstallation",
+    "python -c 'import alphafold'",
+    "python %(installdir)s/bin/run_alphafold_test.py",
+    "alphafold --help 2>&1 | grep 'Full AlphaFold protein structure prediction script'",
+]
+
+sanity_pip_check = True
+
+# these allow to make predictions on proteins that would typically be too long to fit into GPU memory;
+# see https://github.com/deepmind/alphafold/blob/main/docker/run_docker.py
+modextravars = {
+    # these allow to make predictions on proteins that would typically be too long to fit into GPU memory;
+    # see https://github.com/deepmind/alphafold/blob/main/docker/run_docker.py
+    'TF_FORCE_UNIFIED_MEMORY': '1',
+    # jaxlib 0.4.1: https://jax.readthedocs.io/en/latest/changelog.html#jaxlib-0-4-1-dec-13-2022 
+    # "The behavior of XLA_PYTHON_CLIENT_MEM_FRACTION=.XX has been changed to allocate XX% of the total GPU memory 
+    # instead of the previous behavior of using currently available GPU memory to calculate preallocation. Please refer
+    # to GPU memory allocation for more details."
+    # https://jax.readthedocs.io/en/latest/gpu_memory_allocation.html
+    'XLA_PYTHON_CLIENT_MEM_FRACTION': '2.5',
+    # 
+    # Download with $EBROOTALPHAFOLD/scripts/download_all_data.sh /path/to/AlphaFold_DBs/$EBVERSIONALPHAFOLD
+    'ALPHAFOLD_DATA_DIR': '/path/to/AlphaFold_DBs/%(versions)s',  # please adapt
+    'OPENMM_RELAX': 'CUDA'  # unset or set to 'CPU' in order not to run the energy minimization on GPU; PR#189
+}
+
+modloadmsg = """Unlike previous modules, this module is using the updated UniRef30_2023_02 by default!
+In order to use the old UniRef30, please explicitly pass the command line parameter:
+--uniref30_database_path $ALPHAFOLD_DATA_DIR/uniref30/UniRef30_2021_03"""
+
+# Remark: Please modify $EBROOTALPHAFOLD/scripts/download_uniref30.sh in order to download old UniRef30 2021.03
```

Comment:
@ThomasHoffmann77 Any reason why this isn't handled in the patch file `AlphaFold-2.3.2_data-dep-paths-shebang-UniRef30_2023_02.patch` that is used by this easyconfig, could it not just download both?